charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -54,32 +54,8 @@ spec:
             value: "kubernetes.default.svc"
           - name: KUBERNETES_SERVICE_PORT
             value: "443"
-          - name: HELM_SCALAR_DB_CONTACT_POINTS
-            value: "{{ .Values.scalardbCluster.storageConfiguration.contactPoints }}"
-          - name: HELM_SCALAR_DB_CONTACT_PORT
-            value: "{{ .Values.scalardbCluster.storageConfiguration.contactPort }}"
-          - name: HELM_SCALAR_DB_USERNAME
-            valueFrom:
-              secretKeyRef:
-              {{- if .Values.scalardbCluster.existingSecret }}
-                name: {{ .Values.scalardbCluster.existingSecret }}
-              {{- else }}
-                name: {{ include "scalardb-cluster.fullname" . }}-secret
-              {{- end }}
-                key: db-username
-          - name: HELM_SCALAR_DB_PASSWORD
-            valueFrom:
-              secretKeyRef:
-              {{- if .Values.scalardbCluster.existingSecret }}
-                name: {{ .Values.scalardbCluster.existingSecret }}
-              {{- else }}
-                name: {{ include "scalardb-cluster.fullname" . }}-secret
-              {{- end }}
-                key: db-password
-          - name: HELM_SCALAR_DB_STORAGE
-            value: "{{ .Values.scalardbCluster.storageConfiguration.storage }}"
```

Comment:
This environment variable is related to the deprecated `scalardbCluster.storageConfiguration.*` configuration.